### DISCUSSION TITLE
[Feature] Auto album creation via app

### DISCUSSION THREAD

**Discussion by amgschnappi** [immich-app/immich#9298] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I am trying to upload whats app videos, whats app images and camera videos + images.
It would be great, if albums like "wa videos", "wa images", "camera" etc are auto created and the files uploaded in their respective folders.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#1678
